### ISSUE TITLE
Add feature to transfer cash to team mates

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#2127]
Add feature to transfer cash to team mates. Requested by Relax. Would have to be an optional setting for team matches.

**Comment by MTKing4** [comment]
Good Idea, allying and breaking alliances have been part of past C&C games like RA2/YR, was very fun and interesting, allows to have random teammates based on your position on the map

Sharing Units is also a good addition to have, many games have this and it makes things more interesting.
